llvm/lib/Target/RISCV/RISCVRegisterInfo.td

```diff
@@ -664,5 +664,5 @@ def FRM    : RISCVReg<0, "frm">;
 // Shadow Stack register
 def SSP    : RISCVReg<0, "ssp">;
 
-// Dummy VCIX state register
-def VCIX_STATE : RISCVReg<0, "vcix_state">;
+// Dummy SiFive VCIX state register
+def SF_VCIX_STATE : RISCVReg<0, "sf_vcix_state">;
```

Comment:
wait, so why `sf_vcix_state` rather than `sf.vcix_state` here?
https://github.com/riscv-non-isa/riscv-toolchain-conventions/pull/56

Reply:
Umm... I guess I just copied the defining name and converted to lower case..
Let me change it, thanks for catching!